mysite/notes/templates/notes/addnote.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html' %}
+{% block content %}
+  <header  class="header-color-style">
+    <center > Create Note </center>
+  </header>
+  
+</div>
+    
+<div class="row">
+    <div class="col-md-12">
+        <center>
+        {% if error %}
+        <ul>
+            <li>{{ error }}</li>
```

Comment:
rather than using generic error, use form.field.error. 
For more information read here:  https://docs.djangoproject.com/en/2.1/topics/forms/#rendering-form-error-messages